browser-test/password-expired.test.js

```
@@ -34,12 +35,17 @@ describe("Selenium tests for expired password flow />", () => {
     await driver.wait(until.urlContains("change-password"), 5000);
     let successToastDiv = await getElementByCss(driver, "div[role=alert]");
     await driver.wait(until.elementIsVisible(successToastDiv));
-    expect(await successToastDiv.getText()).toEqual("Login successful");
+    
+    
+    const toastText = await waitForToastText(driver, successToastDiv);
+    expect(toastText).toEqual("Login successful");
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)